<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentForm</class>
 <widget class="QWidget" name="DepartmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>추가</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_update">
       <property name="text">
        <string>수정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_init">
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_no">
       <property name="text">
        <string>부서번호</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_no"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_name">
       <property name="text">
        <string>부서명</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_floor">
       <property name="text">
        <string>위치</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_floor"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
